west.yml

```diff
@@ -23,6 +23,8 @@ manifest:
       url-base: https://github.com/zephyrproject-rtos
     - name: babblesim
       url-base: https://github.com/BabbleSim
+    - name: renesas
```

Comment:
Please don't do this. Instead, open a Pull Request in the Zephyr hal_renesas repo and use `pull/<PR number>/head` as the revision.

Reply:
@carlescufi : Thank you so much for your feedback. We fixed it. Please check.